app/mailers/group_upgrade_mailer.rb

```diff
@@ -1,12 +1,12 @@
 class GroupUpgradeMailer < GovukNotifyRails::Mailer
-  def group_upgraded_email(upgraded_by_user:, to_email:, group:, group_url:)
+  def group_upgraded_email(upgraded_by_name:, to_email:, group_name:, group_url:)
```

Comment:
What's the reason behind this refactor, is it for a particular reason or following a particular principle/pattern? It would be good to have it in the commit message.

Reply:
Yeh just following the convention of not passing around more than we need to. Kelvin did a similar refactor recently so thought it was worthwhile following that principle while we were editing this class. We can add something to the commit message.